packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/Models/ProjectedModel.cs

```diff
@@ -2,11 +2,29 @@
 
 #nullable disable
 
+using System;
+
 namespace UnbrandedTypeSpec.Models
 {
     public partial class ProjectedModel
     {
-        // Add Constructors
+        /// <summary> Initializes a new instance of <see cref="ProjectedModel"/>. </summary>
+        /// <param name="name"> name of the ModelWithProjectedName. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="name"/> is null. </exception>
+        public ProjectedModel(string name)
+        {
+            if (name == null)
+            {
+                throw new ArgumentNullException(nameof(name));
+            }
```

Comment:
so we no longer use the `Argument` helper class here. Is this because we do not want it or because we did not implement it yet?

Reply:
Internal types are not implemented yet.